docs/book/user-guide/starter-guide/fetching-pipelines.md

```diff
@@ -101,6 +101,16 @@ Calling a pipeline executes it and then returns the response of the freshly exec
 last_run = training_pipeline()
 ```
 
+{% hint style="warning" %}
+The run that you get back is the model stored in the ZenML database at the point of the method call. This means the pipeline run is still initializing and no steps have been run. To get the latest state can get a refreshed version from the client:
+
+```python
+from zenml.client import Client
+
+Client().get_pipeline_run(last_run.id) to get a refreshed version
```

Comment:
Maybe we should also not call it `last_run`. This isn't actually the last run as someone might have created another one in the meantime, but this is exactly the run that was created by calling `training_pipeline()` in the script above.